spring-web-app/src/test/java/com/epam/viktoryia/springdatabase/ClientRestTemplateTest.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.viktoryia.springdatabase;
+
+import com.epam.viktoryia.springdatabase.model.Employee;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.*;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClientRestTemplateTest extends Mockito {
+
+    private static final Integer ID = 1;
+    private static final String URL = "http://localhost:8080/employee/";
+
+    @Mock
+    RestTemplate restTemplate;
+    @InjectMocks
+    ClientRestTemplate clientRestTemplate;
+    @Captor
+    ArgumentCaptor <HttpEntity> httpEntityCaptor = ArgumentCaptor.forClass(HttpEntity.class);
```

Comment:
Refer, please, to https://www.journaldev.com/21892/mockito-argumentcaptor-captor-annotation
for understanding diff between ArgumentCaptor and @Captor